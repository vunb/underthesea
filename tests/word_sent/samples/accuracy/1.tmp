cộng
hòa
xã hội chủ nghĩa