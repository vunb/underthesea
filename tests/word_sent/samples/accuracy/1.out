cộng
hòa
<<<<<<< HEAD
xã hội chủ nghĩa
=======
xã hội chủ nghĩa
>>>>>>> upstream/master
